yarn.lock

```diff
@@ -10397,6 +10397,36 @@ mocha@7.1.0:
     yargs-parser "13.1.1"
     yargs-unparser "1.6.0"
 
+mocha@^7.1.1:
```

Comment:
what's up with all the changes here?

Reply:
Idk but apparently you're supposed to commit yarn.lock?
https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

Reply:
only if you change package.json though, right?

Reply:
Reverted it.